lib/smack/Naming.cpp

```diff
@@ -66,6 +67,7 @@ const std::string Naming::PTR_VAR = "$p";
 const std::string Naming::UNDEF_SYM = "$u";
 const std::string Naming::CONTRACT_EXPR = "$expr";
 const std::string Naming::MEMORY_SAFETY_FUNCTION = "__SMACK_check_memory_safety";
+const std::string Naming::MEMORY_LEAK_CHECK_FUNCTION = "__SMACK_check_memory_leak";
```

Comment:
Either rename this into MEMORY_LEAK_FUNCTION, or rename the above one into MEMORY_SAFETY_CHECK_FUNCTION. Basically, make these two variable names uniform.